src/python/pants/util/osutil.py

```diff
@@ -72,14 +72,14 @@ def normalize_os_name(os_name: str) -> str:
     return _normalize(os_name, OS_ALIASES, "operating system")
 
 
-def _normalize(name: str, aliases: dict[str, set[str]], warning_hint: str) -> str:
+def _normalize(name: str, aliases: Dict[str, Set[str]], warning_hint: str) -> str:
```

Comment:
If you add `from __future__ import annotations` right under copyright, this isn't necessary